plugins/node/instrumentation-undici/test/undici.test.ts

```diff
@@ -207,7 +210,13 @@ describe('UndiciInstrumentation `undici` tests', function () {
       assert.ok(spans.length === 0, 'ignoreRequestHook is filtering requests');
     });
 
-    it('should create valid spans for different request methods', async function () {
+    it.only('should create valid spans for different request methods', async function () {
```

Comment:
I guess this is a leftover. This `.only` will skip the rest of the tests always

Reply:
Thanks. My bad. commit 2c3ce0ce